src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Changelog.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.Type
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Entity
+data class Changelog(
+  @Column(name = "referral_id") val referralId: UUID,
+
+  @Id
+  @Column(name = "changelog_id")
+  val id: UUID,
+
+  var topic: String,
+
+  @Type(type = "jsonb")
+  @Column(name = "old_value", columnDefinition = "jsonb")
+  val oldVal: String,
```

Comment:
Can't we not map it an object. so that we don't want to map it an object manually